charts/portal/templates/apim/apim-deployment.yaml

```diff
@@ -55,6 +55,17 @@ spec:
         - name: liquidbase-upgrade-rbac
           image: "{{ .Values.global.portalRepository }}{{ .Values.image.upgradeVerify }}"
           args: ["job", 'rbac-upgrade']
+        {{- if .Values.apim.preflightCheck.enabled }}
+        - name: apim-core-preflight-check
+          image: "{{ .Values.global.portalRepository }}{{ .Values.image.upgradeVerify }}"
+          command: ["/bin/sh","-c"]
+          args:
+          - sleep 120;
+            if [[ "$(curl -w '%{http_code}' -s -o /dev/null rabbitmq:15672)" != "200" ]]; then echo "broker not ready" && exit 1;fi;
+            if [[ "$(curl -w '%{http_code}' -s -o /dev/null portal-data:8080/portal-data/portalHealth)" != "200" ]]; then echo "portal-data not ready" && exit 1;fi;
```

Comment:
apim container always had implicit dependency on portal-data and tps containers. so having them here is not required.

Reply:
removed the checks for tps and portal-data